### ISSUE TITLE
fix quicli warning

### ISSUE THREAD

**Issue by ashleygwilliams** [ashleygwilliams/cargo-generate#89]
```
warning: use of deprecated item 'quicli::prelude::Error::causes': please use the 'iter_chain()' method instead
   --> src/main.rs:66:1
    |
66  | / main!(|_cli: Cli| {
67  | |     let args: Args = match Cli::from_args() {
68  | |         Cli::Generate(args) => args,
69  | |         Cli::Gen(args) => args,
...   |
114 | |     );
115 | | });
    | |___^
    |
    = note: #[warn(deprecated)] on by default
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)
```

**Comment by killercup** [comment]
blocked by new quicli release containing https://github.com/killercup/quicli/pull/88

**Comment by ashleygwilliams** [comment]
oh word. i just have a PR with a cargo update and now i get this deprecation warning. good to know!

**Comment by k0pernicus** [comment]
I just checked the code and I don't see where we use `causes` in the main macro...  
Is it related to the imported quicli crate version?

**Comment by killercup** [comment]
@k0pernicus you are probably looking at source of quicli master :) I'm sorry, I still have to release a new version. I wanted it to contain a few other things but then it slipped my mind… I'll re-add it to my todo list and will comment here when it's done. (Otherwise, ping me relentlessly!)

**Comment by k0pernicus** [comment]
@killercup Thank you for the update :)

**Comment by ashleygwilliams** [comment]
just pinged @killercup on this and it should be happening today. @k0pernicus are you interested in taking this issue? let me know!

**Comment by k0pernicus** [comment]
@ashleygwilliams I am :)

**Comment by killercup** [comment]
@ashleygwilliams @k0pernicus here you go: https://crates.io/crates/quicli/0.3.1

**Comment by k0pernicus** [comment]
Thanks @killercup.
I will update the library in less than 1 hour ;)

**Comment by ashleygwilliams** [comment]
closed by #105
